tests/jerry-test-suite/11/11.08/11.08.06/11.08.06-008.js

```diff
@@ -0,0 +1,17 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var a = null;
+var c = a instanceof Object;
+assert (!c);
```

Comment:
Isn't this a bit convoluted way of saying
```js
assert (!(null instanceof Object)
```
?

Reply:
I used the other files from that directory as sample.

Reply:
Thanks for the clarification, now it's fine for me as well.